src/actions/ServiceGraphFilterActions.ts

```diff
@@ -8,17 +8,19 @@ export enum ServiceGraphFilterActionKeys {
   TOGGLE_GRAPH_CIRCUIT_BREAKERS = 'TOGGLE_GRAPH_CIRCUIT_BREAKERS',
   TOGGLE_GRAPH_ROUTE_RULES = 'TOGGLE_GRAPH_ROUTE_RULES',
   // Disable Actions
-  DISABLE_GRAPH_LAYERS = 'DISABLE_GRAPH_LAYERS'
+  ENABLE_GRAPH_FILTERS = 'ENABLE_GRAPH_FILTERS'
 }
 
+const keys = ServiceGraphFilterActionKeys;
```

Comment:
Why is this indirection preferred over the full name?

Reply:
It is totally not needed. I have seen many of you trying to shorten the names so I guess this is me caving in. I actually don't like it.
I say spell everything out. But I know this is not the practice in Go.

Reply:
Then let's spell everything out! Lets not cave in to bad practices.